### PR TITLE
Remove uppercase top nav

### LINKED ISSUE #168: Remove uppercase from top navigation
The text in the word doc needs to be title case then the code needs to remove uppercase transform

line  218
header nav .nav-sections a {
    font-size: var(--heading-font-size-l);
    line-height: 1.25em;
    **font-weight: 700;** take down from 800
    text-decoration: none;
  /*  text-transform: uppercase; */ remove
}

Remove this font weight line 241 and bump up the size
header nav .nav-sections a {
    **font-size: var(--body-font-size-m);** make it m currently at small
    line-height: 1;
    /* font-weight: 500; */ remove lets make them bold
  }

### PR DESCRIPTION
**Pull request by sforschen**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #168

Test URLs:
- Before: https://main--elixirsolutions--hlxsites.hlx.live/
- After: https://remove-uppercase-top-nav--elixirsolutions--hlxsites.hlx.live/

### CODE REVIEW

File: blocks/header/header.css [hunk 1]
```diff
@@ -239,7 +238,6 @@ header nav .nav-sections a {
   header nav .nav-sections a {
     font-size: var(--body-font-size-s);
```
Comment (janefrommaine) [inline blocks/header/header.css:239]: @sforschen, I'm wondering if using a larger font size for desktop might work well here  
--body-font-size-m 
and removing the font-weight
font-weight: unset;

File: blocks/toc/toc.css [hunk 2]
```diff
@@ -1,3 +1,22 @@
...
+  padding-inline: 24px;
+}
+  
+li.toc-h2 {
+  font-weight: 700;
+}
+  
+li.toc-h3 {
+  margin: 16px 0;
+  font-weight: 500;
```
Comment (janefrommaine) [inline blocks/toc/toc.css:21]: @sforschen, in a separate ticket we might want to consider abstracting the font-weights that we use in a css variable so we have an idea of which weights we use and which ones we don't recommend using.

**Review by janefrommaine — CHANGES_REQUESTED** [review]
@sforschen - looking good, I'm mostly wondering if we can bump the header nav's desktop font-size and not use the bold font - so it will match more closely with the Member Portal top nav.
Open to discuss though - just seemed a little off to me.

File: blocks/toc/toc.css (hunk 2 above)
Comment (sforschen) [reply blocks/toc/toc.css:21]: Absolutely, I have already found some variables in the code already. I will add a separate issue about unifying and updating these.

**Comment by sforschen** [comment]
@janefrommaine Unfortunately we cannot make it bigger right now because of the media size. 
I am all for unbolding to match is was just a tad light to me but its not a preference of mine. 

image

**Comment by sforschen** [comment]
@janefrommaine I removed the bolding. Future design I want to change the logo (the one without a tagline) and top nav layout a bit, I am going to include making the nav font larger in that ticket since it will be a little more intensive redesign.

**Comment by janefrommaine** [comment]
@sforschen, oh I see - there are a couple things that need to be tweaked with the main nav.
If you want to add the bold back until we can fix the other issues, that makes sense to me. I see what you mean that the small text + bold might be easier to read while we bridge the gap.
Happy to go with whichever you prefer (small bold vs. small unbold)

One thing I was looking into was:
- larger font
- no bold
- removing the --default-content-gap padding on the navigation (keep it at 1rem) for the (min-width: 56.25em) breakpoint but adding it back for the (min-width: 75em)

Definitely still some alignment issues though.

image

**Comment by sforschen** [comment]
@janefrommaine Know of any way to delete or rescind a commit? If I could delete the bolding one this will be good to go for now.

**Review by janefrommaine — APPROVED** [review]

**Pull request merged**